components/SearchSuggestions/SearchSuggestions.jsx

```diff
@@ -0,0 +1,75 @@
+'use strict'
+
+require('./SearchSuggestions.scss')
+const React = require('react')
+import StandardListItem from '../StandardListItem/StandardListItem.jsx'
+import Panel from '../Panel/Panel'
+
+const SearchSuggestions = {
```

Comment:
convert to ES6 class and put getInitialState in constructor (see comments on SearchBar (https://github.com/appirio-tech/react-components/pull/35/files)